src/main/java/org/craftercms/studio/impl/v2/repository/GitContentRepository.java

```diff
@@ -1354,6 +1354,57 @@ public boolean contentExists(String site, String path) {
         return toReturn;
     }
 
+    @Override
+    public String createFolder(String site, String path, String name) {
+        String commitId = null;
```

Comment:
Please refactor this method to never return null. Throw an exception if there is any error

Reply:
I updated to throw exceptions instead of returning null